InfiniteScrollView.js

```diff
@@ -116,18 +117,23 @@ class InfiniteScrollView extends React.Component {
   }
 
   _distanceFromEnd(event): number {
-    let {
+    const {
```

Comment:
My point was that we want to use `const` for values that are constant across the lifetime of the program, not the lifetime of a function.

Re: performance -- A moderately smart VM would easily be able to see that the variables are assigned to exactly once and there are no direct `eval` calls within this function, allowing the VM to apply all the `const` optimizations. If you can show the performance difference is notable I could be convinced, but I believe the difference is either non-existent or in the noise.